org.kde.falkon.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "org.kde.falkon",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "falkon",
+    "rename-icon": "falkon",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--filesystem=home",
+        "--system-talk-name=org.freedesktop.GeoClue2",
+        "--system-talk-name=org.freedesktop.UPower.*",
+        "--talk-name=org.kde.kwalletd5",
+        "--own-name=org.mpris.MediaPlayer2.*"
```

Comment:
does this need to be so broad? we should know what ID we're going to show up to play music as shouldn't we?

Reply:
Updated to `org.mpris.MediaPlayer2.falkon.*`